packages/pigeon/e2e_tests/test_objc/lib/main.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'dartle.dart';
+
+class _MyFlutterSearchApi extends FlutterSearchApi {
+  @override
+  SearchReply search(SearchRequest input) {
+    return SearchReply()..result = 'Hello ${input.query}, from Flutter';
+  }
+}
+
+void main() {
+  WidgetsFlutterBinding.ensureInitialized();
+  FlutterSearchApiSetup(_MyFlutterSearchApi());
```

Comment:
top level static functions in dart are still in camel case

Reply:
I noticed that too, I plan on changing it, not in this PR.